charts/gardener-extension-provider-aws/templates/service.yaml

```diff
@@ -9,11 +9,17 @@ metadata:
     networking.resources.gardener.cloud/from-policy-allowed-ports: '[{"port":{{ .Values.metricsPort }},"protocol":"TCP"}]'
     networking.resources.gardener.cloud/namespace-selectors: '[{"matchLabels":{"kubernetes.io/metadata.name":"garden"}}]'
     networking.resources.gardener.cloud/pod-label-selector-namespace-alias: extensions
-{{- if .Values.ignoreResources }}
+    {{- if .Values.ignoreResources }}
     resources.gardener.cloud/ignore: "true"
-{{- end }}
+    {{- end }}
+    {{- if .Values.gardener.seed.spec.settings.topologyAwareRouting.enabled }}
+    service.kubernetes.io/topology-aware-hints: "auto"
+    {{- end }}
```

Comment:
While testing this in local setup I noticed that changing the Seed spec (for example enable/disable the topologyAwareRouting setting) does not reconcile the ControllerInstallation (or does not update the Service in automated manner).